File: sgilt-core/src/main/java/net/franzka/sgilt/core/reservation/dto/ProReservationDetailDto.java
package net.franzka.sgilt.core.reservation.dto;

import java.time.LocalDate;
import java.util.UUID;

public record ProReservationDetailDto(
        UUID id,
        String statut,
        String category,
        String prestataireName,
        String prestatairePhoto,
        String evenementTitre,
        String evenementType,
        String evenementImagePath,
        LocalDate evenementDate,
        String evenementVille,
        String clientFirstName,
        String clientLastName,
        String clientPhone,
        String clientEmail
) {}

File: sgilt-front/app/data/reservation/dto/ProReservationDetailDto.ts
export interface ProReservationDetailDto {
  id: string
  statut: string
  category: string
  prestataireName: string
  prestatairePhoto: string | null
  evenementTitre: string
  evenementType: string | null
  evenementImagePath: string | null
  evenementDate: string | null
  evenementVille: string | null
  clientFirstName: string
  clientLastName: string | null
  clientPhone: string | null
  clientEmail: string
}
